Tests/PointFreeTests/AccountTests/UpdateProfileTests.swift

```diff
@@ -96,6 +96,13 @@ class UpdateProfileTests: TestCase {
     assertSnapshot(matching: output, as: .conn)
     #endif
   }
+}
+
+class UpdateProfileTests: LiveDatabaseTestCase {
```

Comment:
should this inherit from `TestCase` instead?

Reply:
Ha yeah.